pkg/cli/admin/inspect/resource.go

```diff
@@ -137,15 +165,11 @@ func gatherRelatedObjects(context *resourceContext, unstr *unstructured.Unstruct
 			errs = append(errs, fmt.Errorf("skipping gathering %s due to error: %v", objectReferenceToString(relatedRef), err))
 			continue
 		}
-
-		for _, relatedInfo := range relatedInfos {
-			if err := InspectResource(relatedInfo, context, o); err != nil {
-				errs = append(errs, fmt.Errorf("skipping gathering %s due to error: %v", objectReferenceToString(relatedRef), err))
-				continue
-			}
-		}
+		allRelatedInfos = append(allRelatedInfos, relatedInfos...)
 	}
 
+	gatherErrs := ParallelInspectResource(allRelatedInfos, context, o)
+	errs = append(errs, gatherErrs...)
```

Comment:
nit: `return errors.NewAggregate(append(errs, gatherErrs...))`

Reply:
or more fancy:

```
return errors.NewAggregate(append(errs, ParallelInspectResource(allRelatedInfos, context, o)...))
```

Reply:
I hate overly complicated statements, this reads cleanly and probably gets optimized by the compiler, so why not making our life's easier :wink: